contracts/exchangeIssuance/ExchangeIssuanceLeveraged.sol

```diff
@@ -28,17 +28,14 @@ import { IAaveLeverageModule } from "../interfaces/IAaveLeverageModule.sol";
 import { IDebtIssuanceModule } from "../interfaces/IDebtIssuanceModule.sol";
 import { IController } from "../interfaces/IController.sol";
 import { ISetToken } from "../interfaces/ISetToken.sol";
+import { IQuoter } from "../interfaces/IQuoter.sol";
```

Comment:
I don't think you are using the `IQuoter` in this contract. Could probably remove the import.